### PR TITLE
Updated canvas scroll + zoom functionality + tab updates

### PR DESCRIPTION
**Pull request by BrianYanGitHub**
Changes:

- Scrollbar scrolls to the bottom of the most recently added element.
- Added 'Back to Top' button to scroll back up to the start of the canvas.
- User now has the ability to zoom in and out of their canvas
- tab has been updated to include ReacType title and icon
-  revised customization panel description in the bottom panel to include 'click to add'
- changed color theme to blue
- allow all elements except img, label and Image to be nestable

### CODE REVIEW

File: app/src/components/main/CanvasContainer.tsx [hunk 1]
```diff
@@ -19,33 +19,74 @@ function CanvasContainer(props): JSX.Element {
...
+      container.scrollTop = 0;
+    }
+  }, [state.components[0].children.length]);
+
   return (
-    <div style={canvasContainerStyle}>
+    <div style={canvasContainerStyle} ref={containerRef}>
       {state.codePreview && <CodePreview theme={theme} setTheme={setTheme} />}
-      {!state.codePreview && <Canvas isThemeLight={props.isThemeLight} />}
+      {!state.codePreview && <Canvas /*isThemeLight={props.isThemeLight} */ />}
```
Comment (johnwage) [inline app/src/components/main/CanvasContainer.tsx:77]: Is `isThemeLight={props.isThemeLight}` deprecated code now?

File: app/src/components/main/Canvas.tsx [hunk 2]
```diff
@@ -306,63 +309,117 @@ const Canvas = (props: {}): JSX.Element => {
...
+    flexDirection: 'column',
+    alignItems: 'center'
+  };
+
+  const buttonStyle: React.CSSProperties = {
+    // position: 'fixed',
+    // width: '10px',
+    // height: '10px',
+    // bottom: '150px',
+    // right: '45vw',
```
Comment (johnwage) [inline app/src/components/main/Canvas.tsx:337]: Sorry, I forgot to remove the comments on lines 333-337.  These can be removed

**Review by johnwage — APPROVED** [review]
Great work, Brian!! I added just a minor comment in `CanvasContainer.tsx` and on `Canvas.tsx` (regarding lines 333-337), but nothing crucial!  We can make those adjustments, if it's not too much trouble.  Looks good to merge otherwise!

**Comment by Eliza612** [comment]
Great work! Looks good and I feel we can merge with the comments removed

**Comment by BrianYanGitHub** [comment]
Done

**Review by johnwage — APPROVED** [review]
Super, Brian!  Reviewed the changes from this morning as well as the nest elements functionality from this evening.  Looking great!

**Pull request merged**